components/ws-daemon/pkg/content/service.go

```diff
@@ -240,6 +241,22 @@ func (s *WorkspaceService) InitWorkspace(ctx context.Context, req *api.InitWorks
 		}
 	}
 
+	if req.PersistentVolumeClaim {
+		// create a folder that is used to store data from running prestophook
+		deamonDir := fmt.Sprintf("%s-daemon", req.Id)
+		prestophookDir := filepath.Join(s.config.WorkingArea, deamonDir, "prestophookdata")
+		_, err = exec.CommandContext(ctx, "mkdir", "-p", prestophookDir).CombinedOutput()
```

Comment:
Perhaps using `os.MkdirAll()` is better to keep consistency.

Reply:
Fixed.